tests/src/Kernel/Access/WikiTest.php

```diff
@@ -184,6 +161,66 @@ public function testEntityOperationAccess($user, array $expected_results) {
   }
 
   /**
+   * Tests exception is thrown when trying to save member role.
+   *
+   * @expectedException \LogicException
+   */
+  public function testMemberRoleMembershipSave() {
+    /** @var \Drupal\og\Entity\OgRole $role */
+    $role = OgRole::create();
+    $role
+      ->setGroupType('block_content')
+      ->setGroupBundle('group')
+      ->setName(OgRoleInterface::AUTHENTICATED)
+      ->setLabel($this->randomString())
+      ->grantPermission('edit any group_content content');
+    $role->save();
+
+    // Subscribe the normal member and the blocked member types to the group.
+    foreach (['member', 'blocked'] as $membership_type) {
+      $state = $membership_type === 'member' ? OgMembershipInterface::STATE_ACTIVE : OgMembershipInterface::STATE_BLOCKED;
```

Comment:
the state isn't important here - the only thing we care about is to make sure you cannot `addRole()` with the member/non-meber roles